scripts/bundleTs.mjs

```diff
@@ -200,8 +200,10 @@ const excludedPackages = new Set(['@jest/globals']);
         filepath,
       });
 
+      const {sep} = path;
+      const replaceDirReg = new RegExp(`(?<=\\${sep})(dist)(?=\\${sep})`);
```

Comment:
@SimenB you'll need to remove this line too

```suggestion
      const {sep} = path;
```

Reply:
yes，i remove it😊